src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -101,6 +100,8 @@ public class ShadowCompareRunsPane extends JPanePlugin {
     private boolean serverHasUpdatedOurRun;
     
     private JPanel dynamicallyRefreshPanel;
+    
+    private String defaultDuration = "5";
```

Comment:
This should be near the top of the class, and declared as a constant:
private static final String DEFAULT_REFRESH_INTERVAL = "5";
This way, it's easy for someone reading the code to find and change.